recipe/meta.yaml

```diff
@@ -33,8 +33,6 @@ requirements:
     - fastapi <1.0.0
     - uvicorn <0.26.0
     - sudachidict-core 20230927
-    - sphinx >=5.1.1,<6.0.0
-    - pydata-sphinx-theme >=0.10.1,<0.11.0
```

Comment:
https://github.com/himkt/konoha/blob/4fa251f3008c5509a33f1af25058c3af57ce9151/pyproject.toml#L40
sphinx and pydata-sphinx-theme is the dependencies of documentation, which are the optional in konoha. I don't assume developers use `conda` for the package development so I think it would be acceptable.

Reply:
Yes that makes sense.

You can also move these into dev dependencies in konoha, and then grayskull should also not pull them in.

This is a good example: https://github.com/aws-powertools/powertools-lambda-python/blob/develop/pyproject.toml#L52

Reply:
Ah I see, that's reasonable. These should be dev dependencies rather than extras...
Thank you, I'll revise pyproject.toml!